packages/lib/services/interop/InteropService_Exporter_Md.ts

```diff
@@ -107,6 +110,10 @@ export default class InteropService_Exporter_Md extends InteropService_Exporter_
 		}
 	}
 
+	async getNoteExportContent_(modNote: NoteEntity) {
+		return await Note.serializeForEdit(modNote);
+	}
+
```

Comment:
Moving this into a function will be useful when adding Yaml functionality because we can inherit from this class and override this function.

Reply:
Could you make it a protected method please?